docs/docker_quickstart.md

```diff
@@ -125,6 +125,11 @@ All freqtrade arguments will be available by running `docker-compose run --rm fr
 
 !!! Note "`docker-compose run --rm`"
     Including `--rm` will remove the container after completion, and is highly recommended for all modes except trading mode (running with `freqtrade trade` command).
+    
+!!! Note that "`docker-compose run --rm`" will require a compose file to be provided.
+Some freqtrade commands that don't require authentication such as `list-pairs` can be run with "`docker run --rm`" instead.
+For example `docker run --rm freqtradeorg/freqtrade:stable list-pairs --exchange binance --quote BTC --print-json`. 
+This can be useful for fetching exchange information to add to your `config.json` without affecting your running containers.
```

Comment:
```suggestion
??? Note "Using docker without docker-compose"
    "`docker-compose run --rm`" will require a compose file to be provided.
    Some freqtrade commands that don't require authentication such as `list-pairs` can be run with "`docker run --rm`" instead.
    For example `docker run --rm freqtradeorg/freqtrade:stable list-pairs --exchange binance --quote BTC --print-json`. 
    This can be useful for fetching exchange information to add to your `config.json` without affecting your running containers.
```

Note boxes require quite specific syntax to work / render correctly ... 
the best in that case is usually to use `mkdocs serve` - and navigate to the localhost version to investigate if the rendering is correct or not.